### ISSUE TITLE
You are making money with these tokens over your own mistake!

### ISSUE THREAD

**Issue by Crispoloko** [stackblitz/bolt.new#2391]
### Describe the bug

I am tired of this. more than 4 millions tokens just for repair inconsistency.

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-dsrfve

### Steps to reproduce

i can't believe it!

### Expected behavior

fix the problem!

### Platform

```
Browser name = Chrome
Full version = 130.0.0.0
Major version = 130
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 85870805,
  "usedJSHeapSize": 76898837,
  "jsHeapSizeLimit": 2160066560
}
Username = Crispo
Chat ID = 56b0adc82a3a
```

### Additional context

give my money back!!!!

**Comment by etheuer** [comment]
agree, we're spending too many millions of token to solve the error it itself generates

**Comment by Crispoloko** [comment]
OMG! Yep! Like us too!I see your point because with this, as you are in the beginning, you can lose credibility.Cristiano EmidioCEO | BackupServices, ***@***.***


On Thu, 14 Nov 2024 at 7:18 PM Erik wrote:

agree, we're spending too many millions of token to solve the error it itself generates

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/stackblitz/bolt.new/issues/2391#issuecomment-2477677708",
"url": "https://github.com/stackblitz/bolt.new/issues/2391#issuecomment-2477677708",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
